docs/misc.rst

```diff
@@ -0,0 +1,203 @@
+            48PB3gXNxzL5/wkv/SZEa65lhbJHo0y/SwsazssrQ3i9p/dlwg6tZZtEFJDK9a7r\
+            MYa3Xq5lbBtYeUU9MTAsX+u5HnEPFLYkzCsKC9pfv7kA+zX37wN5n0e4rAG3AaGp\
+            U9yT/Oy8xKxHao82asi6NM3JIzJlwymk4Kf0F4D+A5hbpAdy9zW1YXovQppb"
+
+
+If you already have a SSH2-formatted public key, you can easily convert it
+to a PKCS8-formatted key for use with Azure using the following command
+(on Fedora 23 - small adjustments may be needed for other systems running
+different versions of OpenSSL).
+
+.. code-block:: bash
+
+    ssh-keygen -f /path/to/linuxvm.pub.pem -i -m PKCS8
+
+
+VirtualMachineExtension Limitations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Microsoft has imposed an odd limitation on running multiple Virtual Machine
+Extensions of the same type (CustomScriptExtension, for instance).  This
+introduces an even further limitation within the scope of a Cloudify
+blueprint using this plugin.  Cloudify Agents for Windows require that
+WinRM be accessible on the target host and this is currently only
+achievable by using a CustomScriptExtension to enable and configure
+WinRM-HTTP (unencrypted, port 5985).  So, since the built in
+cloudify.azure.nodes.compute.VirtualMachine node type has to use
+this Virtual Machine Extension, it means that users will not be able to
+specify a cloudify.azure.nodes.compute.VirtualMachineExtension in their
+blueprint if it is to be of type CustomScriptExtension (since there already
+is one to enable WinRM on the host).  This is not true for Linux as it relies on
+SSH and SSH is enabled on Azure Linux hosts by default (no script needed).
+
+One workaround for this, if you have a strong need to include your own scripts,
+is to override this built-in script.  You can achieve this by changing the
+script name and/or URI list of the node type.  This is what the type defines
+by default:
+
+.. code-block:: yaml
+
+    interfaces:
+      cloudify.interfaces.lifecycle:
+        create: pkg.cloudify_azure.resources.compute.virtualmachine.create
+        configure:
+          implementation: pkg.cloudify_azure.resources.compute.virtualmachine.configure
+          inputs:
+            ps_entry:
+              default: ps_enable_winrm_http.ps1
+            ps_urls:
+              default:
+              - https://server-fqdn/ps_enable_winrm_http.ps1
+        delete: pkg.cloudify_azure.resources.compute.virtualmachine.delete
+
+The inputs ps_entry and ps_urls can be overriden if you create a new node
+type.  It's recommended that, if you do this, you still reference the
+default script within your override script as to still enable WinRM (unless
+you intend to remove or replace this functionality).  Without the default
+script functionality in place, WinRM will not be enabled properly and the
+Cloudify Agent will not be able to connect to the host.
+
+
+Azure Storage Services REST API
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+As of this writing (March 2016) the Microsoft Azure Storage Services API is an
```

Comment:
As this appears to be notes for future development, rather than something we support at the moment, perhaps this whole section should be moved out of the docs entirely (i.e. from `Azure Storage Services REST API` to the end of the file)?